administrator/components/com_localise/Model/PackageModel.php

```diff
@@ -1331,4 +1333,157 @@ public function uploadOtherFile($file, $location)
 
 		return true;
 	}
+
+	/**
+	 * Method to get the HTML output required to update the translations field list by the selected language tag.
+	 *
+	 * @return   string  The data for the jform_translations field.
+	 */
+	public function updateTranslationsList($data)
+	{
+		$params = ComponentHelper::getParams('com_localise');
+		$reftag	= $params->get('reference', '');
```

Comment:
take off one space before =

Reply:
??? i can not to see that one. If posible, please, paste here the right one.

Reply:
Done

Reply:
strange. It shows here above but not in a text file... Forget it. :)